Sage/carbon
# feat(tabs): align tabs with (frozen) design system

Improves the Tabs component to ensure that it complies to the new Design System concepts, mainly via scroll removal in favour of button navigation control

### Proposed behaviour

Disable horizontal scroll and replace with buttons to allow mouse-only users to be able to navigate the tab list, as per the DS

### Current behaviour

Users must grab the scroll bar and drag it around if they wish to view obscured tabs. This can be quite difficult for users who e.g. experience tremours in their hands, and no alternative navigation approach is provided.

### Checklist


#### QA


### Testing instructions

Using the existing `Tabs` Storybook entry (under the `Responsive` section), resize the screen and ensure that the buttons:
- are shown if there are hidden tabs available to scroll to; or
- are hidden if there are no further tabs to show.

Only horizontal tabs are scrollable. Vertical tabs remain unchanged.
